### ISSUE TITLE
Pinephone ARM snap

### ISSUE THREAD

**Issue by magdesign** [NicolasConstant/sengi#371]
Would be amazing to get an ARM build (for ARCH) in the snap store to be able to install/use on Pinephone ;)

**Comment by ahangarha** [comment]
Why not to use it as webapp?

**Comment by NicolasConstant** [comment]
Yes it might be a bit heavy for the Pinephone to host a docker image just for this, but still, a ARM docker build would definitively be helpful. 🙂

**Comment by magdesign** [comment]
You mean [this](https://sengi.nicolas-constant.com/?qt=1618946291796)?

The thing is that I like to delete all the cookies and stuff when closing the browser and using complicated passwords too.
So there would be no use case for the webapp since I would have to login as well every time I open it.

Its not a minor issue, its just an idea/request ;)

**Comment by ahangarha** [comment]
It is good to have dedicated build for ARM. Just I wanted to say one can use Sengi as webapp at least for now.
